src/encoded/static/components/body_map.js

```diff
@@ -126,6 +126,34 @@ const addingClass = (changedClass, matchingString, removeFlag = false) => {
     }
 };
 
+// Set body map diagram colors based on url, when component mounts
+export const initializeBodyMap = (searchQuery) => {
+    const query = new QueryString(searchQuery);
+    const terms = query.getKeyValues(organField);
+    if (terms.length > 0) {
+        terms.forEach((term) => {
+            if (BodyList[term]) {
+                BodyList[term].forEach((bodyClass) => {
+                    addingClass('active', bodyClass);
+                });
+            }
+            if (SystemsList[term]) {
+                document.getElementById(term).classList.add('active');
+            }
+        });
+    }
+};
+
+// Clearing organ and systems key value selections from a url
+export const clearBodyMapSelectionsFromUrl = (originalUrl) => {
+    const parsedUrl = url.parse(originalUrl);
+    const query = new QueryString(parsedUrl.query);
+    query.deleteKeyValue(organField);
```

Comment:
Can organField or SystemField appear multiple times in the URL? If so, I believe this functions removes them just once.

Reply:
They can appear multiple times, but this function removes all matching parameters: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/static/libs/query_string.js#L87